fastlane/Screengrabfile

```diff
@@ -0,0 +1,16 @@
+use_adb_root(false)
+
+app_package_name('org.catrobat.paintroid.test')
```

Comment:
```suggestion
app_package_name('org.catrobat.paintroid')
```
I guess, though I'm not sure.

Reply:
Yeah that needed to be changed. Is changed